config/samples/external_access/mongodb.com_v1_mongodbcommunity_cr.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: mongodbcommunity.mongodb.com/v1
+kind: MongoDBCommunity
+metadata:
+  name: <mongodb-name>
+spec:
+  members: 3
+  type: ReplicaSet
+  version: "4.2.6"
+  replicaSetHorizons:
+  - horizon: <domain-rs-1>:31181
+  - horizon: <domain-rs-2>:31182
+  - horizon: <domain-rs-3>:31183
+  security:
+    tls:
+      enabled: true
+      certificateKeySecretRef:
+        name: mongodb-tls
+      caConfigMapRef:
+        name: ca-config-map
+    authentication:
+      modes: ["SCRAM"]
+  users:
+    - name: my-user
+      db: admin
+      passwordSecretRef: # a reference to the secret that will be used to generate the user's password
+        name: my-user-password
+      roles:
+        - name: clusterAdmin
+          db: admin
+        - name: userAdminAnyDatabase
+          db: admin
+      scramCredentialsSecretName: my-scram
+  additionalMongodConfig:
+    storage.wiredTiger.engineConfig.journalCompressor: zlib
```

Comment:
[nit] can we remove this? I think we left this once in a sample and we keep propagating it in every other sample even if not needed. We should keep it sample files as small as possible

Reply:
Good point. Resolved in latest commit.